pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/JdbiClient.java

```diff
@@ -57,6 +57,21 @@ long insertClient(@Bind("clientName")String clientName,
     Optional<Long> getClientIdByAuth0ClientAndDomain(@Bind("auth0ClientId") String auth0ClientId,
                                                      @Bind("auth0Domain") String auth0Domain);
 
+    @SqlQuery("SELECT "
+            + "     c.client_id, c.client_name, c.auth0_client_id, c.auth0_signing_secret, "
+            + "     c.web_password_redirect_url, c.is_revoked, c.auth0_tenant_id"
+            + "FROM "
+            + "     client c, auth0_tenant t "
+            + "WHERE "
+            + "     c.auth0_client_id = :auth0ClientId and c.auth0_tenant_id = t.auth0_tenant_id"
+            + "     AND t.auth0_domain = :auth0Domain"
+    )
+    @RegisterConstructorMapper(ClientDto.class)
+    Optional<ClientDto> getClientByAuth0ClientAndDomain(
+            @Bind("auth0ClientId") String auth0ClientId,
+            @Bind("auth0Domain") String auth0Domain
+    );
+
```

Comment:
From the https://broadinstitute.atlassian.net/browse/DDP-4134 implementation sketch, I inferred that a combination of `auth0_client_id` and `auth0_domain` is unique. This was reinforced by examining the table definition: `UNIQUE KEY `auth0_tenant_uk` (`auth0_domain`)`.